src/main/java/com/geronimo/model/Message.java

```diff
@@ -0,0 +1,31 @@
+package com.geronimo.model;
+
+import lombok.Data;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.LinkedList;
+import java.util.List;
+
+@Data
+@Entity
+@Table(name = "messages")
+public class Message extends AuditedEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    private Long user_id;
+
+    @NotNull
+    private Long count_of_reblogs;
```

Comment:
I think we need to have a dedicated table for reblogs, as we need to track the user who did the reblog somehow. Same for likes, we need to know that this particular use has already liked a message or/and reblogged.

this is a Many-to-Many relationship